docs/about.md

```diff
@@ -8,11 +8,13 @@ slug: /
 
 ## Political advocacy
 
-Starting in [November of 2020](https://youtu.be/93leuGiUSpo), I attempted to spin up an organization to mobilize youth into direct, local political action. Our first efforts were in Georgia, where we [knocked on over 20,000](https://www.wrbl.com/news/georgia-news/math-mvmt-hosts-canvassing-event-ahead-of-senate-run-offs/) doors in support of Jon Ossoff and Reverend Warnock. For my next effort, I wanted to focus on my hometown of Omaha, Nebraska to see if we could mobilize enough volunteers to get a progressive candidate elected to office. Unfortunately because of how people and news organizations [mischaracterized my actions](https://www.reddit.com/r/Destiny/comments/lx0dnw/incredibly_disappointed_at_the_lack_of/), I was forced to stop campaigning for my mayor of choice. I've also interviewed a number of individuals ( [1](https://www.youtube.com/watch?v=9AoEIeVrbWU) | [2](https://www.youtube.com/watch?v=aLQUuui7ThA) | [3](https://www.youtube.com/watch?v=M9rtW5Vre2s) | [4](https://www.youtube.com/watch?v=ag23SLc2fIs) ) running for state and local offices while [exposing fraud](https://www.youtube.com/watch?v=J0FnS1gFV4o) in other candidates.
+Starting in [November of 2020](https://youtu.be/93leuGiUSpo), I attempted to mobilize my community, primarily young adults, into direct, local political action. Our first efforts were in Georgia, where we [knocked on over 20,000](https://www.wrbl.com/news/georgia-news/math-mvmt-hosts-canvassing-event-ahead-of-senate-run-offs/) doors in support of Jon Ossoff and Reverend Warnock. For my next effort, I wanted to focus on my hometown of Omaha, Nebraska to see if we could mobilize enough volunteers to get a progressive candidate elected to office. Sadly, because of how I was [mischaracterized online](https://www.reddit.com/r/Destiny/comments/lx0dnw/incredibly_disappointed_at_the_lack_of/) by news organisations and people on social media, my community and I were forced to stop campaigning for our mayor of choice. I've also interviewed a number of individuals ( [1](https://www.youtube.com/watch?v=9AoEIeVrbWU) | [2](https://www.youtube.com/watch?v=aLQUuui7ThA) | [3](https://www.youtube.com/watch?v=M9rtW5Vre2s) | [4](https://www.youtube.com/watch?v=ag23SLc2fIs) ) running for state and local offices while [exposing fraud](https://www.youtube.com/watch?v=J0FnS1gFV4o) in other candidates.
```

Comment:
```suggestion
Starting in [November of 2020](https://youtu.be/93leuGiUSpo), I attempted to mobilize my community, primarily young adults, into direct, local political action. Our first efforts were in Georgia, where we [knocked on over 20,000](https://www.wrbl.com/news/georgia-news/math-mvmt-hosts-canvassing-event-ahead-of-senate-run-offs/) doors in support of Jon Ossoff and Reverend Warnock. For my next effort, I wanted to focus on my hometown of Omaha, Nebraska to see if we could mobilize enough volunteers to get a progressive candidate elected to office. Sadly, because of how I was [mischaracterized online](https://www.reddit.com/r/Destiny/comments/lx0dnw/incredibly_disappointed_at_the_lack_of/) by news organizations and people on social media, my community and I were forced to stop campaigning for our mayor of choice. I've also interviewed a number of individuals ( [1](https://www.youtube.com/watch?v=9AoEIeVrbWU) | [2](https://www.youtube.com/watch?v=aLQUuui7ThA) | [3](https://www.youtube.com/watch?v=M9rtW5Vre2s) | [4](https://www.youtube.com/watch?v=ag23SLc2fIs) ) running for state and local offices while [exposing fraud](https://www.youtube.com/watch?v=J0FnS1gFV4o) in other candidates.
```